salt/_runners/metalk8s_saltutil.py

```diff
@@ -1,7 +1,12 @@
 from __future__ import absolute_import, print_function, unicode_literals
+import logging
 
+import salt.client
 import salt.utils.extmods
 
+LOGGER = logging.getLogger(__name__)
```

Comment:
Salt habits is to call this `log` (yes, unlike PEP8 dictates), but OK :)

Reply:
I see that we use this convention in other Salt modules, so I'll comply with it.